celery/app/log.py

```diff
@@ -153,8 +155,8 @@ def _configure_logger(self, logger, logfile, loglevel,
             if loglevel:
                 logger.setLevel(loglevel)
 
-    def setup_task_loggers(self, loglevel=None, logfile=None, format=None,
-                           colorize=None, propagate=False, **kwargs):
+    def setup_task_loggers(self, loglevel: int | None = None, logfile=None, format=None,
+                           colorize=None, propagate: bool = False, **kwargs):
```

Comment:
Since this is also not part of the change... Feel free to remove it. 👀 
```suggestion
    def setup_task_loggers(self, loglevel=None, logfile=None, format=None,
                           colorize=None, propagate=False, **kwargs):
```